src/prompt/base.rs

```diff
@@ -105,6 +106,10 @@ pub trait Prompt: Send {
     fn get_prompt_color(&self) -> Color {
         DEFAULT_PROMPT_COLOR
     }
+    /// Get the default right prompt color
+    fn get_prompt_multiline_color(&self) -> nu_ansi_term::Color {
+        DEFAULT_PROMPT_MULTILINE_COLOR
+    }
```

Comment:
Added a new function to the `Prompt` to allow for it fetch a color. For now it uses a `nu_ansi_term::Color` unlike the others.